<?xml version="1.0"?>
<!-- 
Ajay jain
July 7, 2014
ARSENL Lab, Naval Postgraduate School
-->
<launch>
  	<group ns="robot_0">
		<node pkg="husky_pursuit" type="offset_pursue.py" name="offset_pursue_node" output="screen">
			<remap from="/cmd_vel" to="/robot_0/cmd_vel"/>
			<remap from="/own_odom" to="/robot_0/odom"/>
			<remap from="/target_odom" to="/robot_1/odom"/>

			<param name="linear_vel_max" value=".8" />
			<param name="angular_vel_max" value="1" />
			<!-- <param name="extrapolation_time" value="5" /> -->
			<param name="extrapolate_with_constant_time" type="boolean" value="false" />

			<!-- <param name="own_tf_prefix" value="/robot_0" /> -->
			<!-- <param name="target_tf_prefix" value="/robot_1" /> -->
		</node>
	</group>



  	<group ns="robot_1">
		<include file="$(find husky_teleop)/launch/teleop.launch">
			<arg name="cmd_topic" value="/robot_1/cmd_vel"/>
		</include>
	</group>

  	<!-- <group ns="robot_2">
		<node pkg="husky_pursuit" type="pure_flee.py" name="flee_node">
			<remap from="/cmd_vel" to="/robot_2/cmd_vel"/>

			<param name="linear_vel_max" value="0.5" />
			<param name="angular_vel_max" value="1" />
			<param name="own_tf_prefix" value="/robot_2" />
			<param name="target_tf_prefix" value="/robot_1" />
		</node>
	</group> -->
</launch>